std_msgs/Header header
float32[] circle3D_center
float32[] circle3D_normal
float32 circle3D_radius
float32 confidence
float32 timestamp
float32 diameter_3d
float32[] ellipse_center
float32[] ellipse_axes
float32 ellipse_angle
float32[] location
float32 diameter
float32[] sphere_center
float32 sphere_radius
float32[] projected_sphere_center
float32[] projected_sphere_axes
float32 projected_sphere_angle
float32 model_confidence
float32 model_id
float32 model_birth_timestamp
float32 theta
float32 phi
float32[] norm_pos
uint32 id
string method
